DependencyInjection/Configuration.php

```diff
@@ -49,15 +61,17 @@ public function getConfigTreeBuilder()
                                     ->arrayNode('roots')
                                         ->isRequired()
                                         ->requiresAtLeastOneElement()
+                                        ->useAttributeAsKey('name')
                                         ->prototype('array')
                                             ->children()
                                                 ->scalarNode('driver')
                                                     ->isRequired()
-                                                    ->defaultValue('LocalFileSystem')->end()
+                                                    ->defaultValue('LocalFileSystem')
+                                                ->end() // driver
                                                 ->arrayNode('disabled')
```

Comment:
can you please also give an example of disabled config (yaml)? I'm unsure if I understand what it has to do. In the current way, it doesn't support XML